environment.yml

```diff
@@ -0,0 +1,22 @@
+name: flu
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+- flask
```

Comment:
note: check if it is the same as https://github.com/conda-forge/fludashboard-feedstock/blob/master/recipe/meta.yaml#L24

if it is not the same, consider to change it here or if your changes is necessary, consider to open a PR to update the feedstock.